src/single-spa-vue.js

```diff
@@ -121,6 +121,12 @@ function mount(opts, mountedInstances, props) {
 
     mountedInstances[props.name] = instance;
 
+    // attach Vue instance to the vue devtools
+    if (window.__VUE_DEVTOOLS_GLOBAL_HOOK__) {
+      window.__VUE_DEVTOOLS_GLOBAL_HOOK__.Vue =
+        instance.vueInstance.constructor;
```

Comment:
I think this just be a reference to `opts.Vue`, right?

```suggestion
      window.__VUE_DEVTOOLS_GLOBAL_HOOK__.Vue = opts.Vue;
```